lib/src/main/kotlin/com/lemonappdev/konsist/api/declaration/KoTypeDeclaration.kt

```diff
@@ -56,6 +56,11 @@ interface KoTypeDeclaration : KoNamedDeclaration {
      */
     val isNullable: Boolean
 
+    /**
+     * Whatever type is basic Kotlin type.
```

Comment:
"Whatever type is a build in Kotlin type. It can be a basic Kotlin type (https://kotlinlang.org/docs/basic-types.html) or collection type (https://kotlinlang.org/docs/collections-overview.html#collection)."

Also update docs in KoTypeDeclarationSequenceExt.kt